api/golang/core/lib/enclaves/enclave_context.go

```diff
@@ -345,6 +345,20 @@ func (enclaveCtx *EnclaveContext) DownloadFilesArtifact(ctx context.Context, art
 	return fileContent, nil
 }
 
+func (enclaveCtx *EnclaveContext) InspectFilesArtifact(ctx context.Context, artifactName services.FileArtifactName) (*kurtosis_core_rpc_api_bindings.InspectFilesArtifactContentsResponse, error) {
```

Comment:
You could potentially expose both - to either get all the file artifacts contents or just one;; though this is can be done later as well.